dodola/tests/test_services.py

```diff
@@ -253,9 +266,18 @@ def test_regrid_weights_integration(tmpdir):
     ds_in = grid_global(30, 20)
     ds_in["fakevariable"] = wave_smooth(ds_in["lon"], ds_in["lat"])
 
+    # make fake domain file
+    lon_name = "lon"
+    lat_name = "lat"
+    domain = grid_global(1, 1)
+    domain = domain.rename({"x": lon_name, "y": lat_name})
+    domain[lat_name] = np.unique(domain[lat_name].values)
+    domain[lon_name] = np.unique(domain[lon_name].values)
```

Comment:
Minor suggestion-lite:

This "make a domain file" logic is being repeated a lot. Creating a domain file Dataset is candidate for a pytest fixture. It'd remove this repetition.